bulldozer/merge.go

```diff
@@ -195,3 +197,30 @@ func allCommits(ctx context.Context, pullCtx pull.Context, client *github.Client
 
 	return repositoryCommits, nil
 }
+
+func postCommentIfUserActionable(ctx context.Context, client *github.Client, owner, repo string, prNum int, githubError github.ErrorResponse) {
+	commentBody := "Error occurred when attempting merge: " + githubError.Message
+
+	switch {
+	case strings.HasPrefix(githubError.Message, "You're not authorized to push to this branch."):
+		commentBody = commentBody + "\n\nThe bulldozer bot account may not be allowed to merge the branch."
```

Comment:
As far as I know, this is specifically when branch protection sets restrictions on who can push to a branch and there are no known workarounds. To me, the current wording suggests you can give Bulldozer more permissions to fix this.

Reply:
yea, the wording should be changed to use a different approach, since bulldozer cant be given more permissions